.github/workflows/build.yml

```diff
@@ -18,18 +23,18 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Validate Gradle Wrapper
-        uses: gradle/wrapper-validation-action@v1
+        uses: gradle/wrapper-validation-action@v1.0.4
 
       - name: Configure JDK
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v3.1.0
```

Comment:
No need to update the revision version, use major version for the latest by default.